modules/external_state/external-state.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$id": "https://app.runx.dev/modules/external-state",
+  "type": "object",
+  "description": "Adds the ability to refer to external, preexisting terraform states in opta.",
+  "properties": {
+    "backend_type": {
+      "type": "string",
+      "description": "The type of backend this external state is using"
```

Comment:
can we also add the link here https://www.terraform.io/language/settings/backends/configuration#backend-types